airbyte-api/src/main/openapi/config.yaml

```diff
@@ -1402,6 +1402,30 @@ paths:
           $ref: "#/components/responses/NotFoundResponse"
         "422":
           $ref: "#/components/responses/InvalidInputResponse"
+  /v1/state/create_or_update:
```

Comment:
Should we document the JWT auth here? Swagger has support for `securitySchemes`

Reply:
I would prefer not to since we override these definitions in [Cloud](https://github.com/airbytehq/airbyte-cloud/blob/master/tools/infra-deploy-tools/common/config-api-sw20-overrides.yaml#L1).

If we add a securitySchema, it's not useful today since the OSS server doesn't actually enforce these. I think we add this when we do the project to bring the user and workspace models over into OSS and start enforcing these even in OSS.

Reply:
moved discussion to child PR here: https://github.com/airbytehq/airbyte/pull/15902/files#r953190405